penelope_aerospace_pl_msgs/msg/TriggersState.msg

```diff
@@ -0,0 +1,13 @@
+#-------------------------------------------------------------------------------
+# DVM Triggers State (TNO)
+#-------------------------------------------------------------------------------
+
+# Description of the DVM process states:
+#   GIVE_TRIGGERS
+#       ...
+#   STOP_GIVING_TRIGGERS
+#       ...
```

Comment:
I would suggest using a boolean value for this in the action definition, something like:
```
# Indicate if the DVM acquisition triggers are active 
#   True if a motion request with triggers for the DVM system is sent to the manipulator
#   and is being processed. False in all other situations.

bool triggers_active
```

If you intend to have more states, you could keep this message, but I would add the other states and update the naming as the verb seems to imply this is requesting triggers. 

I would suggest the following:
```suggestion
#   TRIGGERS_ACTIVE
#        A motion request with triggers for the DVM system is sent to the manipulator
#        and is being processed.
#   NO_TRIGGERS
#       No motion request with triggers is active or being processed. This is a valid
#       state for motions without triggers or while no motion is requested. 
```

Reply:
That's ok for me

Reply:
I fixed the code in the penelope_dvm_module to use the TRIGGERS_ACTIVE and NO_TRIGGERS